### PR TITLE
[chore] Fix mongodb integration test

### LINKED ISSUE #17070: [MongoDB receiver] Flakey integration test skipped
## Description

As a result of resolving https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17037 , the MongoDB receiver had been merged to include an update that fails the integration tests. 

It is currently being skipped to help unblock adding tests back into the workflow, see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/17068

See https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3709312536/jobs/6287813742

Output:

```
make[1]: Entering directory '/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/mongodbreceiver'
running go integration test ./... + coverage in /home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/mongodbreceiver
=== RUN   TestMongodbIntegration
=== RUN   TestMongodbIntegration/Running_mongodb_4.0
=== PAUSE TestMongodbIntegration/Running_mongodb_4.0
=== RUN   TestMongodbIntegration/Running_mongodb_4.2
=== PAUSE TestMongodbIntegration/Running_mongodb_4.2
=== RUN   TestMongodbIntegration/Running_mongodb_4.4_as_LPU
=== PAUSE TestMongodbIntegration/Running_mongodb_4.4_as_LPU
=== RUN   TestMongodbIntegration/Running_mongodb_5.0
=== PAUSE TestMongodbIntegration/Running_mongodb_5.0
=== CONT  TestMongodbIntegration/Running_mongodb_4.2
=== CONT  TestMongodbIntegration/Running_mongodb_4.4_as_LPU
2022/12/16 01:13:21 github.com/testcontainers/testcontainers-go - Connected to docker: 
  Server Version: 20.10.21+azure-2
  API Version: 1.41
  Operating System: Ubuntu 22.04.1 LTS
  Total Memory: 6943 MB
2022/12/16 01:13:21 Starting container id: dc99d61027be image: docker.io/testcontainers/ryuk:0.3.4
2022/12/16 01:13:21 Waiting for container id dc99d61027be image: docker.io/testcontainers/ryuk:0.3.4
2022/12/16 01:13:22 Container is ready id: dc99d61027be image: docker.io/testcontainers/ryuk:0.3.4
2022/12/16 01:13:35 Starting container id: 05e99ceaf6f0 image: 598ecf80-a60f-4e36-8898-5f55de29b48e:1488115b-f7ef-4019-bb43-208fc99c6d1c
2022/12/16 01:13:35 Waiting for container id 05e99ceaf6f0 image: 598ecf80-a60f-4e36-8898-5f55de29b48e:1488115b-f7ef-4019-bb43-208fc99c6d1c
2022/12/16 01:13:36 Container is ready id: 05e99ceaf6f0 image: 598ecf80-a60f-4e36-8898-5f55de29b48e:1488115b-f7ef-4019-bb43-208fc99c6d1c
2022/12/16 01:13:37 Starting container id: 533f70e613a2 image: 73b7fb30-2a52-4[471](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3709312536/jobs/6287813742#step:6:472)-b8e5-fee15e68a9a7:f80d6ec8-d828-4600-9e5c-c178fb1156d0
2022/12/16 01:13:37 Starting container id: 05e99ceaf6f0 image: 598ecf80-a60f-4e36-8898-5f55de29b48e:1488115b-f7ef-4019-bb43-208fc99c6d1c
2022/12/16 01:13:37 Waiting for container id 05e99ceaf6f0 image: 598ecf80-a60f-4e36-8898-5f55de29b48e:1488115b-f7ef-4019-bb43-208fc99c6d1c
2022/12/16 01:13:37 Container is ready id: 05e99ceaf6f0 image: 598ecf80-a60f-4e36-8898-5f55de29b48e:1488115b-f7ef-4019-bb43-208fc99c6d1c
2022/12/16 01:13:37 Waiting for container id 533f70e613a2 image: 73b7fb30-2a52-4471-b8e5-fee15e68a9a7:f80d6ec8-d828-4600-9e5c-c178fb1156d0
2022/12/16 01:13:38 Container is ready id: 533f70e613a2 image: 73b7fb30-2a52-4471-b8e5-fee15e68a9a7:f80d6ec8-d828-4600-9e5c-c178fb1156d0
2022/12/16 01:13:43 Starting container id: 533f70e613a2 image: 73b7fb30-2a52-4471-b8e5-fee15e68a9a7:f80d6ec8-d828-4600-9e5c-c178fb1156d0
2022/12/16 01:13:43 Waiting for container id 533f70e613a2 image: 73b7fb30-2a52-4471-b8e5-fee15e68a9a7:f80d6ec8-d828-4600-9e5c-c178fb1156d0
2022/12/16 01:13:43 Container is ready id: 533f70e613a2 image: 73b7fb30-2a52-4471-b8e5-fee15e68a9a7:f80d6ec8-d828-4600-9e5c-c178fb1156d0
=== CONT  TestMongodbIntegration/Running_mongodb_4.2
    integration_test.go:154: 
        	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/mongodbreceiver/integration_test.go:154
        	Error:      	Received unexpected error:
        	            	number of resources does not match expected: 10, actual: 0
        	Test:       	TestMongodbIntegration/Running_mongodb_4.2
=== CONT  TestMongodbIntegration/Running_mongodb_4.0
=== CONT  TestMongodbIntegration/Running_mongodb_5.0
2022/12/16 01:14:49 Starting container id: 11fb139c36e4 image: 1fbb12bc-ae3c-47e9-8c30-599b65855d2a:0c3588f0-32c3-431e-be55-5711ecd30df9
2022/12/16 01:14:49 Waiting for container id 11fb139c36e4 image: 1fbb12bc-ae3c-47e9-8c30-599b65855d2a:0c3588f0-32c3-431e-be55-5711ecd30df9
2022/12/16 01:14:50 Container is ready id: 11fb139c36e4 image: 1fbb12bc-ae3c-47e9-8c30-599b65855d2a:0c3588f0-32c3-431e-be55-5711ecd30df9
2022/12/16 01:14:50 Starting container id: 11fb139c36e4 image: 1fbb12bc-ae3c-47e9-8c30-599b65855d2a:0c3588f0-32c3-431e-be55-5711ecd30df9
2022/12/16 01:14:50 Waiting for container id 11fb139c36e4 image: 1fbb12bc-ae3c-47e9-8c30-599b65855d2a:0c3588f0-32c3-431e-be55-5711ecd30df9
2022/12/16 01:14:50 Container is ready id: 11fb139c36e4 image: 1fbb12bc-ae3c-47e9-8c30-599b65855d2a:0c3588f0-32c3-431e-be55-5711ecd30df9
2022/12/16 01:15:00 Starting container id: f0fdc9e8ca2f image: dfc67b89-8831-46e5-aadb-c4ba3670bc12:8179ddd4-9764-47ce-8032-6e365f0f6e4c
2022/12/16 01:15:00 Waiting for container id f0fdc9e8ca2f image: dfc67b89-8831-46e5-aadb-c4ba3670bc12:8179ddd4-9764-47ce-8032-6e365f0f6e4c
2022/12/16 01:15:01 Container is ready id: f0fdc9e8ca2f image: dfc67b89-8831-46e5-aadb-c4ba3670bc12:8179ddd4-9764-47ce-8032-6e365f0f6e4c
2022/12/16 01:15:01 Starting container id: f0fdc9e8ca2f image: dfc67b89-8831-46e5-aadb-c4ba3670bc12:8179ddd4-9764-47ce-8032-6e365f0f6e4c
2022/12/16 01:15:01 Waiting for container id f0fdc9e8ca2f image: dfc67b89-8831-46e5-aadb-c4ba3670bc12:8179ddd4-9764-47ce-8032-6e365f0f6e4c
2022/12/16 01:15:01 Container is ready id: f0fdc9e8ca2f image: dfc67b89-8831-46e5-aadb-c4ba3670bc12:8179ddd4-9764-47ce-8032-6e365f0f6e4c
=== CONT  TestMongodbIntegration/Running_mongodb_4.0
    integration_test.go:118: 
        	Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/receiver/mongodbreceiver/integration_test.go:118
        	Error:      	Received unexpected error:
        	            	number of metrics does not match expected: 10, actual: 9
        	Test:       	TestMongodbIntegration/Running_mongodb_4.0
--- FAIL: TestMongodbIntegration (0.00s)
    --- FAIL: TestMongodbIntegration/Running_mongodb_4.2 (77.30s)
    --- PASS: TestMongodbIntegration/Running_mongodb_4.4_as_LPU (85.05s)
    --- FAIL: TestMongodbIntegration/Running_mongodb_4.0 (73.69s)
    --- PASS: TestMongodbIntegration/Running_mongodb_5.0 (76.60s)
FAIL
coverage: 72.8% of statements
FAIL	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mongodbreceiver	161.692s
```

### PR DESCRIPTION
**Pull request by djaglowski**
Resolves #17070

Based on the failure messages I have seen, the test appears to fail due to missing data. My guess here is that the Mongo API is able to respond before the system has fully started, resulting in an incomplete response. This attempts to compensate for such a scenario by discarding the first scrape result.

### CODE REVIEW

File: Makefile.Common [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # In order to ensure make instructions fail if there is command that fails a pipe (ie: `go test ... | tee -a ./test_results.txt`)
```
Comment (mx-psi) [inline Makefile.Common:1]: I think we can revert the changes in this file, right? (not against fixing whitespace, just not related)

**Review by mx-psi — APPROVED** [review]

File: Makefile.Common (hunk 1 above)
Comment (djaglowski) [reply Makefile.Common:1]: Whoops, unintentional. Good catch.
